code-reviews/recipes/Python.md

```diff
@@ -1,53 +1,208 @@
 # Python Code Reviews
 
-## Python Style Guide
+## Style Guide
 
-[CSE](../CSE.md) developers follow [Google's Python Style Guide](https://google.github.io/styleguide/pyguide.html). Note the use of strong typing throughout. Older versions of python allow you to get away without this but type checking avoids common errors that are tricky to debug. 
+[CSE](../../CSE.md) developers follow the [PEP8 style guide](https://pep8.org/) with [type hints](https://www.python.org/dev/peps/pep-0484/). The use of type hints throughout paired with linting and type hint checking avoids common errors that are tricky to debug.
 
-## Linters
+[CSE](../../CSE.md) projects should check Python code with automated tools.
 
-[CSE](../CSE.md) projects should check Python code with automated tools. ```pylint``` is the minimum, but we prefer to run three different tools:
+Linting should be added to build validation, and both linting and code formatting can be added to your pre-commit hooks and VS Code.
```

Comment:
Same as my comment on Bash - if linting should be part of CI, it should be added tot he CI document.

Reply:
If it is not already part of the CI/CD documentation and not reported as an issue I think it would probably be good to add it.  Will you? or should I?